va/va_dec_hevc.h

```diff
@@ -194,6 +194,163 @@ typedef struct  _VAPictureParameterBufferHEVC
+ *
+ * This structure conveys picture level HEVC Range Extension parameters
+ * and should be sent once per frame. This data structure should be sent
+ * together with VAPictureParameterBufferHEVC in a single buffer since each frame
+ * of HEVC range extension contains both picture level parameters and picture
+ * level range extension parameters. They should be parsed together.
+ *
+ */
+typedef struct  _VAPictureParameterBufferHEVCRext
+{
+    union
+    {
+        struct
+        {
+            /** \brief HEVC range extension flags
+             *  The following flags bears the same syntax and semantics as
+             *  those defined in HEVC bit stream spec.
+             */
+            uint32_t        transform_skip_rotation_enabled_flag        : 1;
+            uint32_t        transform_skip_context_enabled_flag         : 1;
+            uint32_t        implicit_rdpcm_enabled_flag                 : 1;
+            uint32_t        explicit_rdpcm_enabled_flag                 : 1;
+            uint32_t        extended_precision_processing_flag          : 1;
+            uint32_t        intra_smoothing_disabled_flag               : 1;
+            uint32_t        high_precision_offsets_enabled_flag         : 1;
+            uint32_t        persistent_rice_adaptation_enabled_flag     : 1;
+            uint32_t        cabac_bypass_alignment_enabled_flag         : 1;
+            uint32_t        cross_component_prediction_enabled_flag     : 1;
+            uint32_t        chroma_qp_offset_list_enabled_flag          : 1;
+
+            /** \brief Reserved bytes for future use, must be zero */
+            uint32_t        reserved_bits                               : 21;
+        } bits;
+        uint32_t            value;
+    } range_extension_pic_fields;
+
+    /** \brief HEVC range extension flags
+     *  The following flags bears the same syntax and semantics as
+     *  those defined in HEVC bit stream spec.
+     */
+    uint8_t                 diff_cu_chroma_qp_offset_depth;
+    uint8_t                 chroma_qp_offset_list_len_minus1;
+    uint8_t                 log2_sao_offset_scale_luma;
+    uint8_t                 log2_sao_offset_scale_chroma;
+    uint8_t                 log2_max_transform_skip_block_size_minus2;
+    int8_t                  cb_qp_offset_list[6];
+    int8_t                  cr_qp_offset_list[6];
+} VAPictureParameterBufferHEVCRext;
+
+/**
+ *\brief HEVC Decoding Picture Parameter Buffer Structure for
+ *Screen Content extension
+ *
+ *This structure conveys picture level HEVC Scc parameters
+ *and should be sent once per frame. This data structure should be sent
+ *together with VAPictureParameterBufferHEVC and VAPictureParameterBufferHEVCRext
+ *in a single buffer since each frame of HEVC SCC contains picture level
+ *parameters, picture level range extension parameters and picture level
+ *Scc parameters. They should be parsed together.
```

Comment:
Could you add the doc about VASliceParameterBufferHEVCExtension here or a reference to VASliceParameterBufferHEVCExtension via doxygen command in the comment? the buffer type used for this single buffer should be documented as well.